components/Counters/Counters.js

```diff
@@ -42,7 +49,26 @@ function Counters({ boxes, buckets, setBoxes, setBuckets }) {
           />
         ))}
       </CountersContainer>
-    </div>
+      <CountersContainer>
+        {attachments.map((value, index) => (
+          <Counter
+            key={index}
+            name={`Aufsätze ${index === 0 ? "geliefert:" : "zurück:"}`}
+            value={value}
+            onIncrement={() => {
+              const attachmentsCopy = [...attachments];
+              attachmentsCopy[index] += 1;
+              setAttachments(attachmentsCopy);
+            }}
+            onDecrement={() => {
+              const attachmentsCopy = [...attachments];
+              attachmentsCopy[index] -= 1;
+              setAttachments(attachmentsCopy);
+            }}
```

Comment:
Your solution works fine, so this is just my personal opinion: 
Why do you do that complicated stuff with mapping over that array. Since you just have an array with two entries (0 and 0) I would prefer to render two Counter-Components:

<Counter name="Aufsätze geliefert" value={0} onIncrement={setAttachments((prev)=>prev+1)} onDecrement={setAttachments((prev)=>prev-1)} />
<Counter name="Aufsätze zurück" value={0} onIncrement={setAttachments((prev)=>prev-1)} onDecrement={setAttachments((prev)=>prev-1)} />

With this solution you won't need the key-prop anymore (and even the value can be set inside the Counter) and things become a bit easier.

Reply:
Yes youre right, its confusing like this. For first ive just changed the key thing, because i´d like to concentrate on some other things id like to finish before its over-